docs/customization.md

```diff
@@ -28,6 +28,8 @@ Here are the props that `MegadraftEditor` accepts:
   and when clicked it will open a modal window with the full button list.
 - `modalOptions`: (optional) object, height and width of the modal.
   Check the following sections for more info.
+- `onToolbarShowRequest`: (optional) Predicate function fired when
+  editor state changes
```

Comment:
Hey, sorry to jump in, but I had some comments. 

I think this functionality is pretty useful for having better control over when a toolbar shows up and in fact this is peripherally useful for something I'm trying to do. :)

I think the documentation needs a bit more clarity. Ultimately this is a function used to determine whether the toolbar should show or not when it is mounted. As written, it's clear when this is used, but not why. Also, it's assumed that this function returns a boolean (because it's used in a conditional). This also needs to be explicit.

I wonder also if this could benefit from a different name. Maybe "shouldToolbarShowFn"? Kind of tricky.